src/components/Home.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable react/react-in-jsx-scope */
+import React from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+import ListTask from './ListTasks';
+
+const styles = StyleSheet.create({
+  mainContainer: {
+    flex: 1,
+    color: 'red',
+  },
+  header: {
+    backgroundColor: '#rgb(31,134,255)',
```

Comment:
Same here.

Also try to standarize the way you define a color, we usually define them in `hexadecimal` format